/* @generated */
digraph cfg {
"junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_1" [label="1: Start void Failure.runTest()\nFormals:  this:junit.tests.framework.Failure*\nLocals:  \n  " color=yellow style=filled]
	

	 "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_1" -> "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_4" ;
"junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_2" [label="2: Exit void Failure.runTest() \n  " color=yellow style=filled]
	

"junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_3" [label="3:  exceptions sink \n   APPLY_ABSTRACTION; [line 12]\n " shape="box"]
	

	 "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_3" -> "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_2" ;
"junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_4" [label="4:  Call void TestCase.fail() \n   n$0=_fun_void TestCase.fail()() [line 11]\n  EXIT_SCOPE(n$0); [line 11]\n " shape="box"]
	

	 "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_4" -> "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_5" ;
	 "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_4" -> "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_3" [color="red" ];
"junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_5" [label="5:  method_body \n   APPLY_ABSTRACTION; [line 12]\n " shape="box"]
	

	 "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_5" -> "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_2" ;
	 "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_5" -> "junit.tests.framework.Failure.runTest():void.9244e3395ba81efe8c61970dbe0f8b1f_3" [color="red" ];
}
